DuckDuckGo/NetworkProtectionVPNLocationView.swift

```diff
@@ -28,73 +28,172 @@ struct NetworkProtectionVPNLocationView: View {
 
     var body: some View {
         List {
-            Text("⚠️ FEATURE IS WORK IN PROGRESS ⚠️")
-            Section {
-                Button(action: model.onNearestItemSelection) {
-                    Text(UserText.netPPreferredLocationNearest)
-                }
-            }
-            Section {
-                ForEach(model.countryItems) { item in
-                    Button(action: {
-                        model.onCountryItemSelection(countryID: item.countryID)
-                    }, label: {
-                        Text(item.localizedName)
-                    })
-                }
+            switch model.state {
+            case .loading:
+                EmptyView()
```

Comment:
This state felt a bit weird for me the first time though - what do you think about an activity indicator instead?